src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { combineReducers, createStore, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import  booksReducer  from './books/books';
+
+const reducer = combineReducers({
+  booksReducer
+});
```

Comment:
- [ ] Please do not forget to import the Categories Reducer and add it to the `combineReducer` method.